src/cli_config.rs

```diff
@@ -98,35 +98,35 @@ pub struct EncodeArgs {
 
 #[derive(Debug, Clone, Parser)]
 pub struct DecodeArgs {
-    /// the JWT to decode
+    /// The JWT to decode. Provide '-' to read from STDIN.
     #[clap(index = 1)]
     #[clap(value_parser)]
-    pub jwt: String,
+    pub jwt: Option<String>,
```

Comment:
I'm not sure why this was changed to be an `Option`? As part of the command line arguments, it should be present/required

Reply:
Oops, that was a mistake. Fixed!